a3p-integration/proposals/g:resolve-usdc/package.json

```
@@ -0,0 +1,34 @@
+{
+  "agoricProposal": {
+    "source": "subdir",
+    "sdk-generate": [
+      "../../portfolio-deploy/src/access-token-setup.build.js submission",
+      "../../portfolio-deploy/src/chain-info.build.js submission --net devnet --peer noble:connection-13:channel-11:uusdc --peer axelar:connection-19:channel-315:uaxl",
```

Getting dynamic devnet config (using `agd`) isn't appropriate here.

Other core-evals have an A3P_INTEGRATION config, but that doesn't make sense here: a3p-integration has no connection to a noble chain. We won't be able to test contract functionality in this environment. But our goal here is to test contract deployment itself: just getting the contract into `published.agoricNames.instance`.

So just use the default (mainnet) config:

```suggestion
      "../../portfolio-deploy/src/chain-info.build.js submission",
```